src/Host/Broker/Impl/About/AboutInfo.cs

```diff
@@ -5,27 +5,21 @@
 using System.Linq;
 using System.Management;
 using System.Reflection;
-using Microsoft.AspNetCore.Authorization;
-using Microsoft.AspNetCore.Mvc;
 using Microsoft.R.Host.Broker.Interpreters;
-using Microsoft.R.Host.Broker.Security;
 using Microsoft.R.Host.Broker.Sessions;
 using Microsoft.R.Host.Protocol;
 
 namespace Microsoft.R.Host.Broker.About {
-    [Authorize(Policy = Policies.RUser)]
-    [Route("/info/about")]
-    public class AboutController : Controller {
-        private readonly InterpreterManager _interpManager;
-        private readonly SessionManager _sessionManager;
+    public class AboutInfo {
```

Comment:
It should either be a static class, or a service. I think the latter would be preferable (you can then get components injected via constructor etc).